frontend/src/components/plugins/utils/utils.jsx

```diff
@@ -186,6 +191,61 @@ export function PluginHealthCheckButton({ pluginName, pluginType }) {
   );
 }
 
+export function OrganizationPluginStateToggle({
+  disabled,
+  pluginName,
+  type,
+  refetch,
+}) {
+  const {
+    isUserOwner,
+    organization,
+    fetchAll: fetchAllOrganizations,
+  } = useOrganizationStore(
+    React.useCallback(
+      (state) => ({
+        fetchAll: state.fetchAll,
+        isUserOwner: state.isUserOwner,
+        organization: state.organization,
+      }),
+      []
+    )
+  );
+  let title = "";
+  if (!organization.name) title = "You're not a part of any organization";
+  else if (!isUserOwner)
+    title = `You're not an owner of your organization - ${organization.name}`;
+
+  return (
+    <div className="d-flex flex-column align-items-center" title={title}>
+      <InputCheckBox
```

Comment:
what I meant is that this should be a button and not an checkbox because checkbox usually do not contain actions while buttons do and in this case if you click you change a state so it should be a button. Does it make sense now?

Reply:
Done :+1: